src/app/Caches/Create/FeaturesSelector.tsx

```diff
@@ -59,20 +63,24 @@ const FeaturesSelector = (props: {
             indexedCache: indexedCache,
             securedCache: securedCache,
             backupsCache: backupsCache,
-            transactionalCache: transactionalCache
+            transactionalCache: transactionalCache,
+            persistentCache: persistentCache
         });
 
         let validForm = true;
         if (cacheFeatureSelected.length > 0) {
-            validForm = boundedFeatureValidation(validForm)
-            validForm = indexingFeatureValidation(validForm);
-            validForm = securedFeatureValidation(validForm);
-            validForm = backupsFeatureValidation(validForm);
-            validForm = transactionalFeatureValidation(validForm);
+            const boundedValid = boundedFeatureValidation(validForm);
+            const indexingValid = indexingFeatureValidation(validForm);
+            const securedValid = securedFeatureValidation(validForm);
+            const backupsValid = backupsFeatureValidation(validForm);
+            const transactionalValid = transactionalFeatureValidation(validForm);
+            const persistentValid = persistentFeatureValidation(validForm);
+
+            validForm = boundedValid && indexingValid && securedValid && backupsValid && transactionalValid && persistentValid;
```

Comment:
@karesti Made some changes in feature validation, now if any of the feature validation is false, the final would be false.